packages/ui/src/Chips/ChipIcon.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { ChipIcon as RmwcChipIcon, ChipIconProps as RmwcChipIconProps } from "@rmwc/chip";
+import { Chip as RmwcChipIcon, ChipProps as RmwcChipIconProps } from "@rmwc/chip";
```

Comment:
Previously, there were 2 separate components: a `Chip`, and a `ChipIcon`. `ChipIcon` was used as a child of the `Chip` component to add an icon to a chip, here's an example:

```
<Chip>
    <ChipIcon icon={<AnIcon/>}/>
    Chip Label
</Chip>
```

This was changed ages ago, but this `<ChipIcon>` component was not removed. We don't use it anywhere in Webiny, so feel free to delete this component and stories for it. It's meaningless because the `<Chip>` itself supports `icon` props.

Reply:
Done, thanks